test/rocc/rocc-test.c

```diff
@@ -0,0 +1,48 @@
+// This test program uses the example ROCC accelerator that is included with
+// spike by default (called dummy-rocc).
+// Use the provided Makefile to build
+// Run with spike --extension=dummy_rocc rocc-test.riscv
+
+#include <stdio.h>
+#include <stdint.h>
+#include "rocc.h"
+
+// Old custom format:
```

Comment:
I'm not sure I understood this comment

Reply:
Whoops! That was a note-to-self from when I was converting this code to Schuylers macros. I've removed it.